JohnsonCountyJail.yaml

```diff
@@ -1313,251 +1263,431 @@ associationDefinitions:
     src: "ArrestingAgency"
     dst: "JohnsonCounty_IowaCity_JCCharge"
-    entitySetName: "JohnsonCounty_IowaCity_JCInvolvedIn"
     propertyDefinitions:
       ol.id:
-        type: ol.id
-        column: Jail_ID
+        type: "ol.id"
+        column: "Jail_ID"
     name: "JohnsonCounty_IowaCity_JCInvolvedIn1"
+
+  JohnsonCounty_IowaCity_JCInvolvedIn10:
+    fqn: "ol.involvedin"
+    entitySetName: "JohnsonCounty_IowaCity_JCInvolvedIn"
+    src: "Aofficerperson"
+    dst: "JohnsonCounty_IowaCity_JCCharge"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "Jail_ID"
+      ol.role:
+        type: "ol.role"
+        transforms:
+        - !<transforms.ValueTransform>
+          value: "Officer"
+      ol.roledescription:
+        type: "ol.roledescription"
+        transforms:
+        - !<transforms.ValueTransform>
+          value: "Arresting"
+    name: "JohnsonCounty_IowaCity_JCInvolvedIn10"
+
   JohnsonCounty_IowaCity_JCInvolvedIn2:
     fqn: "ol.involvedin"
+    entitySetName: "JohnsonCounty_IowaCity_JCInvolvedIn"
     src: "TransportingAgency"
     dst: "JohnsonCounty_IowaCity_JCJailStay"
-    entitySetName: "JohnsonCounty_IowaCity_JCInvolvedIn"
     propertyDefinitions:
       ol.id:
-        type: ol.id
-        column: Jail_ID
+        type: "ol.id"
+        column: "Jail_ID"
     name: "JohnsonCounty_IowaCity_JCInvolvedIn2"
+
+  JohnsonCounty_IowaCity_JCInvolvedIn3:
+    fqn: "ol.involvedin"
+    entitySetName: "JohnsonCounty_IowaCity_JCInvolvedIn"
+    src: "Tofficerperson"
+    dst: "JohnsonCounty_IowaCity_JCJailStay"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "Jail_ID"
+      ol.role:
+        type: "ol.role"
+        transforms:
+        - !<transforms.ValueTransform>
+          value: "Officer"
+      ol.roledescription:
+        type: "ol.roledescription"
+        transforms:
+        - !<transforms.ValueTransform>
+          value: "Transporting"
+    name: "JohnsonCounty_IowaCity_JCInvolvedIn3"
+
   JohnsonCounty_IowaCity_JCInvolvedIn4:
     fqn: "ol.involvedin"
+    entitySetName: "JohnsonCounty_IowaCity_JCInvolvedIn"
     src: "ReleasingAgency"
     dst: "JohnsonCounty_IowaCity_JCJailStay"
-    entitySetName: "JohnsonCounty_IowaCity_JCInvolvedIn"
     propertyDefinitions:
       ol.id:
-        type: ol.id
-        column: Jail_ID
+        type: "ol.id"
+        column: "Jail_ID"
     name: "JohnsonCounty_IowaCity_JCInvolvedIn4"
+
   JohnsonCounty_IowaCity_JCInvolvedIn5:
     fqn: "ol.involvedin"
+    entitySetName: "JohnsonCounty_IowaCity_JCInvolvedIn"
     src: "ChargingAgency"
     dst: "JohnsonCounty_IowaCity_JCCharge"
-    entitySetName: "JohnsonCounty_IowaCity_JCInvolvedIn"
     propertyDefinitions:
       ol.id:
-        type: ol.id
-        column: Jail_ID
+        type: "ol.id"
+        column: "Jail_ID"
     name: "JohnsonCounty_IowaCity_JCInvolvedIn5"
+
   JohnsonCounty_IowaCity_JCInvolvedIn6:
```

Comment:
same comment as above - we should use the ol.employee entity set instead of the general.person entity set for the officers in these